app/services/concerns/pdf_builder/active_storage.rb

```diff
@@ -0,0 +1,21 @@
+module PdfBuilder
```

Comment:
The pdf builders share some common code.  Rather than having a superclass that they both inherit from which didn't seem right (its like saying all pdf's will work this way - which we dont really know) - I decided to break down chunks into concerns and just include them